src/lib/y2storage/md_parity.rb

```diff
@@ -29,5 +29,39 @@ class MdParity
     include StorageEnumWrapper
 
     wrap_enum "MdParity"
+
+    TRANSLATIONS = {
+      # TODO: should be tested by some real OPS guy, I (JR) pick human wording from `man mdadm`
```

Comment:
Thanks god we have git blame. Otherwise, it would be hard to find who JR is. :) My first though was:

![JR](https://i.ytimg.com/vi/GZV3365a7Ew/hqdefault.jpg)